### PR TITLE
Feature/mobileye testing

### PR DESCRIPTION
**Pull request by msmcconnell**
This adds the launch file I used for mobileye testing to this repo and creates a calibration file with the same serial number as the Mako camera used for that testing. Also corrects the video resolution and sets the IP to a value usable in the cadillacs.

### CODE REVIEW

File: calibrations/calibration_50-0536911040.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Calibration file for the spare mobileye camera at STOL
```
Comment (qswawrq) [inline calibrations/calibration_50-0536911040.yaml]: It looks like a calibration file from mako camera, not mobileye?

File: launch/mobileye_eval.launch [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!--
```
Comment (qswawrq) [inline launch/mobileye_eval.launch]: I am not sure where we should keep this file. Feel like does not belong this repo. Do you think it makes sense to put it in our engineering tool folder?

File: calibrations/calibration_50-0536911040.yaml (hunk 1 above)
Comment (msmcconnell) [reply calibrations/calibration_50-0536911040.yaml]: Oops yeah that's a typo

File: launch/mobileye_eval.launch (hunk 2 above)
Comment (msmcconnell) [reply launch/mobileye_eval.launch]: I think it is best if we keep it with a package so we can run 
roslaunch <pkg> mobileye_eval.launch easily. 

I could move it to the CARMAPlatform repo if you want. There are some other testing launch files I have used in the past there. Let me know.

**Comment by maefromm** [comment]
@qswawrq @msmcconnell  Do you want to keep this PR left open, or can this be resolved?

**Comment by qswawrq** [comment]
> @qswawrq @msmcconnell Do you want to keep this PR left open, or can this be resolved?

I think it is fine to merge this branch.

**Pull request merged**